test/Error/aggregate_error.js

```diff
@@ -0,0 +1,384 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Copyright (c) 2021 ChakraCore Project Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+var tests = [
+   {
+       name: "proptotype and constructor",

```

Comment:
Please can we have slightly more descriptive test names e.g. "AggregateError Prototype and Constructor Shape"